data/map systems.txt

```diff
@@ -5924,10 +5925,10 @@ system Aspidiske
 
 system Atik
 	pos -66 -52
-	government Uninhabited
+	government "Humanika (Restricted)"
 	attributes "bright star" "giant star" "multi-star" "notable star"
-	arrival 5000
-	habitable 15105
+	arrival 1800
```

Comment:
```suggestion
	arrival 3105
```
I may be wrong about this, but iirc systems with a habitable range between 500 and 5000 have arrival distances equal to the habitable range.

Reply:
That is too large. They don't interact enough.

Reply:
There is nothing for most of the ships to do in the system since most ships can't land at La'ab Bamen. Most of the ships leave immediately unless there's someone to shoot at. The appearance I want to give is that ships are going into the system to "monitor things" (except miners and Independent pilots). For that to work, they have to be within 5000 pixels when they arrive, so the "you are too far for me to care" logic doesn't kick in.

Reply:
They're both 1800 now. Is that good?

Reply:
That's an issue everywhere else too, and should probably have been looked at when the arrival distances were first added. This works as a fix for now, but at some point we should adjust the AI's detection range to fix this.

Reply:
Revising the habitable ranges and arrival distances of all systems is not within the scope of this PR. The only one I want to touch is Atik, since it is the only one where the combat has changed.